package.json

```diff
@@ -17,15 +17,16 @@
     "examples": "examples"
   },
   "scripts": {
-    "lint": "eslint gulpfile.js src/*.js src/**/*.js",
-    "build": "gulp",
+    "lint": "eslint src/*.js src/**/*.js",
+    "start": "DEV_SERVER=true node scripts/dev-server",
+    "build": "./scripts/build.sh",
+    "rollup": "rollup --config",
```

Comment:
Do we need a rollup script?  Haven't yet seen if you're using it below.  Def fine if so.

Reply:
and ... i see the usage.  nothing to see here ;)